IdentityCore/tests/integration/ios/MSIDBrokerTokenRequestTests.m

```diff
@@ -176,7 +176,7 @@ - (void)testInitBrokerRequest_whenParametersWithOptionalParameters_shouldReturnV
                                       @"redirect_uri": @"my-redirect://com.microsoft.test",
                                       @"broker_key": @"brokerKey",
                                       @"client_version": [MSIDVersion sdkVersion],
-                                      @"extra_qp": @"my_eqp1%2C+%2C=my_eqp2&my_eqp3=my_eqp4",
+                                      @"extra_qp": @"my_eqp1%2C%20%2C=my_eqp2&my_eqp3=my_eqp4",
```

Comment:
To double check: if we deal with old broker accepting request from new ADAL (with those changes), will old broker be able to parse extra qp encoded this way.

Reply:
Will change this back, as we agreed to not change adal-broker communication and do form encoding.